### PR TITLE
Fix yeoman generator usage doc.

### PR DESCRIPTION
**Pull request by zbigg**

### CODE REVIEW

File: @here/generator-harp.gl/README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Yeoman generator for [harp.gl](https://github.com/heremaps/harp.gl) based projec
 ```sh
 mkdir 3dmap-example
 cd 3dmap-example
+npm install @here/generator-harp.gl
 yo @here/harp.gl
```
Comment (haraldF) [inline @here/generator-harp.gl/README.md]: `npx` missing? Shouldn't that be `npx yo`?

File: @here/generator-harp.gl/README.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ Yeoman generator for [harp.gl](https://github.com/heremaps/harp.gl) based projec
 ```sh
 mkdir 3dmap-example
 cd 3dmap-example
+npm install @here/generator-harp.gl
```
Comment (haraldF) [inline @here/generator-harp.gl/README.md]: How about `npm install --no-save --no-lock` ?

**Comment by zbigg** [comment]
retest this please

**Comment by zbigg** [comment]
retest this please

**Comment by zbigg** [comment]
retest this please

**Comment by zbigg** [comment]
@heremaps-bot retest this please

**Pull request merged**